src/app/app.module.ts

```diff
@@ -56,43 +28,15 @@ import { JoinSessionComponent } from './pages/join-session/join-session.componen
 import { PrepareRoundTestComponent } from './test-pages/prepare-round-test/prepare-round-test.component';
 import { SessionLobbyComponent } from './pages/session-lobby/session-lobby.component';
 import { BottomBarComponent } from './components/bottom-bar/bottom-bar.component';
-import { HillBackgroundComponent } from './components/hill-background/hill-background.component';
-import {
-  StudentDisplayComponent,
-  StudentRemovedDialogComponent
-} from './pages/student-display/student-display.component';
+import { StudentRemovedDialogComponent } from './pages/student-display/student-display.component';
```

Comment:
Do we want to declare the `StudentRemovedDialogComponent` into the `StudentDisplayModule`? I know that we don't need to lazy-load `StudentRemovedDialogComponent`, since it's really small, but I still kind of want to keep it with the AR stuff, if only for organizational purposes! :)

Reply:
(In particular, it feels a bit strange to me for two classes to be defined in the same file, but included in different modules. I might just be making that up, though! 🙃)

Reply:
Oh crap I just have forgotten to move that. It totally belongs in the studentdisplay module.